src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -84,6 +84,19 @@ public void addMessage(Message message) {
 
     return messagesInConversation;
   }
+  /** Access the current set of Messages from the given author. */
+  public List<Message> getMessagesFromAuthor(UUID authorId) {
+
+    List<Message> messagesFromAuthor = new ArrayList<>();
+    
+    for(Message message : messages) {
+      if(message.getAuthorId().equals(authorId)) {
+	messagesFromAuthor.add(message);
```

Comment:
Seems like there's a little indentation issue here. This line uses tabs instead of spaces? :)

Reply:
will fix.